src/internal/functions/Update-PuppetModuleMetadata.ps1

```diff
@@ -90,6 +90,12 @@ function Update-PuppetModuleMetadata {
     )
     # Clarify Puppet lower bound
     $PuppetMetadata.requirements[0].version_requirement = '>= 6.0.0 < 7.0.0'
+    # Tag the module as coming from the builder so that it becomes searchable on the Forge.
+    $PuppetMetadata | Add-Member -MemberType NoteProperty -Name tags -Value @(
+      'windows',
+      'puppetdscbuilder',
```

Comment:
So the PowerShell module name is set as `puppet.dsc` - should this reflect that name or should we use a special tag like `puppetized_dsc`? 🤔

Reply:
Either `puppet-dsc` or `puppetdsc` seems reasonable to me. (the `.` would be filtered out.)